feat(android): Update wizard and getting-started with the SAGP auto-installation feature (#4836)
getsentry/sentry-docs: src/includes/getting-started-install/android.mdx

> ### Automatically with the Sentry Android Gradle plugin

Reviewer: ```suggestion
### Auto-Installation With the Sentry Android Gradle Plugin
```

> ### Auto-Installation With the Sentry Android Gradle Plugin

Reviewer 2: Worth adding a note that this is only for >= 3.1.0

Author: True. Actually am not sure should we add it here since it's gonna be a preview cc @bruno-garcia ?

To install the plugin, add it to your `build.gradle` file:

```groovy
buildscript {
  repositories {
    mavenCentral()
  }
}
plugins {
  id "io.sentry.android.gradle" version "3.1.0-alpha.1"
}
```

```kotlin
buildscript {
  repositories {
    mavenCentral()
  }
}
plugins {
  id("io.sentry.android.gradle") version "3.1.0-alpha.1"
}
```

> The plugin will automatically add the latest version of the Android SDK as a dependency.

Reviewer 3: ```suggestion
The plugin will automatically add the latest version of the Sentry SDK for Android as a dependency.
```

### Manual Installation

> To install the Android SDK, add it to your `build.gradle` file:

Reviewer 3: Suggestion for rewording so it's more clear that this is an option, other than the Gradle one:

```suggestion
If you don't want the Sentry Gradle plugin to install the Sentry SDK automatically for you, you can define the dependency directly to your `build.gradle` file:
```
